code/game/gamemodes/objective.dm

```diff
@@ -1019,3 +1022,25 @@ GLOBAL_LIST_EMPTY(possible_items)
 	var/area/target_area = get_area(target)
 
 	return (istype(user_area, dropoff) && istype(target_area, dropoff))
+
+
+// SKYRAT EDIT ADDITION BEGIN - ANTAG OPT IN (Mind list for opt in blacklist)
+
+//opt in list, gathers all crew with their settings set to whatever antagonism level we're looking for + all of command and sec
+//set get anyone for true if the antags can target anyone. for contractors and heretics we set this to false b/c they only target sec and command
+/proc/minimum_opt_in_level(level = NOT_TARGET, get_anyone = TRUE)
```

Comment:
this proc can be modularized (taken out of the core file)

i suggest making a new module - see module_template.dm and other modules for examples of how theyre setup, but its usually just (modular_skyrat/modules/yourfolder)

Reply:
global defines go in code/_DEFINES/~skyrat_defines.dm

Reply:
preferences can go wherever

Reply:
(global means used in multiple files)